src/vector/v.what.rast.label/v.what.rast.label.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+
+##############################################################################
+#
+# MODULE:       v.what.rast.label
+# AUTHOR(S):    Paulo van Breugel <paulo at ecodiv dot earth>
+# PURPOSE:      Upload raster values and labels at positions of vector points
+#               to the vector attribute table.
+#
+# COPYRIGHT: (C) 2016-2022 by Paulo van Breugel and the GRASS Development Team
+#
+#        This program is free software under the GNU General Public
+#        License (>=v2). Read the file COPYING that comes with GRASS
+#        for details.
+##############################################################################
+
+#%module
```

Comment:
Should't here everywhere a white space?

```
# %module
# % description:
...
```

Reply:
Now you mention it @neteler , I remember having read about that somewhere. I am happy to change it. However, it is not mentioned on https://trac.osgeo.org/grass/wiki/Submitting/Python as far as I have seen. Moreover, there is an example on that page without those white spaces. So, also because of the question mark in your comment, just checking, should I change it?

Reply:
> Now you mention it @neteler , I remember having read about that somewhere. I am happy to change it. However, it is not mentioned on https://trac.osgeo.org/grass/wiki/Submitting/Python as far as I have seen.

I guess an oversight...

> Moreover, there is an example on that page without those white spaces. So, also because of the question mark in your comment, just checking,

Sorry for the rhetoric question mark which wasn't needed.

> should I change it?

Yes, please change it in the Submitting document.

Reply:
> Yes, please change it in the Submitting document.
done